knime_extension/src/util/knime_utils.py

```diff
@@ -769,3 +769,13 @@ def get_env_path():
 #         ]["weight"]
 
 #     return adjust_list_ref
+
```

Comment:
Can you add a placeholder parameter to the method which contains the default value "your api key here" which should also be checked in addition to empty